frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -75,6 +75,18 @@ class ChartSettings extends Component {
         return assocIn(this.props.series, [0, "card", "visualization_settings"], this.state.settings);
     }
 
+    getChartTypeName(){
+      switch(this.props.series[0].card.display){
+        case "table":
+          return "table";
+        case "scalar":
+          return "number";
+        default:
+          return "chart";
```

Comment:
Yeah we should look into that. I'm inclined to make the default 4 and then scope it if we don't like it for certain filetypes. (this of course all assumes you're using the plugin for your IDE, otherwise chaos prevails)